### DISCUSSION TITLE
Google Spaces enhancement request for alerts/notifications

### DISCUSSION THREAD

**Discussion by tommyreilly** [fluxcd/flux2#4442] [General]
Wondering if there may be any future possibility for the community to consider adding Google Spaces support for flux/flagger alerts, akin to the Slack alerting/notification current capabilities.

**Comment by stefanprodan** [comment]
Wasn't this product killed by Google long time ago? https://killedbygoogle.com

**Reply by tommyreilly** [reply]
I’m referring to the current feature named Spaces within Google Chat that was previously referred to as Rooms, not the old/deprecated Google Spaces mobile app product that I think you may be referring to @stefanprodan 

See:

https://en.m.wikipedia.org/wiki/Google_Chat#Spaces

**Reply by stefanprodan** [reply]
We do support Google Chat https://fluxcd.io/flux/components/notification/providers/#google-chat

**Comment by tommyreilly** [comment]
Thanks @stefanprodan.

Apologies, I should have read that. Appreciate it.

How about Flagger (or should I ask that in a separate forum/channel)?

**Reply by tommyreilly** [reply]
What I’m specifically referring to is the doc at:

https://docs.flagger.app/usage/alerting#

“The alert provider type can be: slack, msteams, rocket or discord”

but does not mention Google Chat/Spaces.

**Reply by stefanprodan** [reply]
Please open an issue on the Flagger repo for Google Chat.

**Reply by tommyreilly** [reply]
Thanks for your advice @stefanprodan 

Opened https://github.com/fluxcd/flagger/issues/1561

Closing this discussion
